app/src/main/java/com/example/concordia_campus_guide/Adapters/RoutesAdapter.java

```diff
@@ -0,0 +1,220 @@
+package com.example.concordia_campus_guide.Adapters;
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import com.example.concordia_campus_guide.ClassConstants;
+import com.example.concordia_campus_guide.Models.Routes.Bus;
+import com.example.concordia_campus_guide.Models.Routes.Car;
+import com.example.concordia_campus_guide.Models.Direction;
+import com.example.concordia_campus_guide.Models.Routes.Route;
+import com.example.concordia_campus_guide.Models.Routes.Subway;
+import com.example.concordia_campus_guide.Models.Routes.Train;
+import com.example.concordia_campus_guide.Models.Routes.TransportType;
+import com.example.concordia_campus_guide.Models.Routes.Walk;
+import com.example.concordia_campus_guide.R;
+import com.google.android.flexbox.FlexboxLayout;
+
+import java.util.List;
+import androidx.annotation.NonNull;
+
+import org.w3c.dom.Text;
+
+public class RoutesAdapter extends ArrayAdapter<Route> {
```

Comment:
Can you add a comment to indicate that this is an adapter for the UI?

Reply:
done